atlas/map.go

```diff
@@ -23,9 +23,10 @@ import (
 	"github.com/go-spatial/geom/planar/makevalid/hitmap"
 	"github.com/go-spatial/geom/planar/simplify"
 	"github.com/go-spatial/geom/slippy"
+	"github.com/go-spatial/proj"
 	"github.com/go-spatial/tegola"
 	"github.com/go-spatial/tegola/dict"
-	"github.com/go-spatial/tegola/proj"
+	tegolaProj "github.com/go-spatial/tegola/proj"
```

Comment:
This alias is necessary because you're referring to the `proj` package constant `proj.WebMercator`. Rather than needing the alias, I propose you create package level constants in the `tegola/proj` package called `WebMercatorSRID` and `WGS84SRID` which point to the `go-spatial/proj` package. This would allow us to keep the interactions with the `go-spatial/proj` package in a single place in the codebase and drop this alias. This suggestion applies to the entire PR.

Reply:
Awesome, thanks @ARolek !  Would this suggestion also apply to the rest of the project (eg. consolidate WGS84 / WebMercator references to be internal to `github.com/go-spatial/tegola/proj` throughout) or is the main intent to drop the alias?